bodhi/tests/server/test_util.py

```diff
@@ -349,28 +351,29 @@ def test_get_critpath_components_dummy(self):
         """
         self.assertEqual(util.get_critpath_components(), ['kernel', 'glibc'])
 
-    @mock.patch.object(pkgdb2client.PkgDB, 'get_critpath_packages')
-    @mock.patch.dict(util.config, {
-        'critpath.type': 'pkgdb',
-        'pkgdb_url': 'http://domain.local'
-    })
-    def test_get_critpath_components_pkgdb_success(self, mock_get_critpath):
-        """ Ensure that critpath packages can be found using PkgDB.
-        """
-        # A subset of critpath packages
-        critpath_pkgs = [
-            'pth',
-            'xorg-x11-server-utils',
-            'giflib',
-            'basesystem'
-        ]
-        mock_get_critpath.return_value = {
-            'pkgs': {
-                'f20': critpath_pkgs
+    if six.PY2:
```

Comment:
Should we use the ```@skipif``` decorator here too?

Reply:
No, because in this case, we need to skip also all the decorators and therefore we cannot use `@unittest.skipif`.